Gruntfile.js

```diff
@@ -64,12 +64,17 @@ module.exports = function (grunt) {
                         color8: '#ab7e5b',
                         color9: '#5b755d'
                     },
+                    icons: {
+                        alert: 'alert',
+                        flag: 'flag',
+                        blah: 'blah'
```

Comment:
just added these to generate some some html for styling, they will be remove when I get around to build a proper demo version